libr/io/io_fd.c

```diff
@@ -126,3 +126,21 @@ R_API int r_io_fd_get_current(RIO *io) {
 	}
 	return -1;
 }
+
+R_API int r_io_fd_get_next(RIO *io, int fd) {
+	r_return_val_if_fail (io, -1);
+	int ret = fd;
+	if (!r_id_storage_get_next (io->files, (ut32 *)&ret)) {
+		return -1;
+	}
+	return ret;
+}
+
+R_API int r_io_fd_get_prev(RIO *io, int fd) {
+	r_return_val_if_fail (io, -1);
+	int ret = fd;
+	if (!r_id_storage_get_prev (io->files, (ut32 *)&ret)) {
```

Comment:
fds in idstorage are ut32, but in io are int?

Reply:
yes, it's intended that way.